opm/parser/eclipse/Deck/Section.hpp

```diff
@@ -47,7 +47,7 @@ class Section : public DeckView {
 
         // returns whether the deck has all mandatory sections and if all sections are in
         // the right order
-        static bool checkSectionTopology(const Deck& deck,
```

Comment:
What do you mean? If you do `MessageContainer& mc = deck->getMessageContainer()` mc is non-const i.e. mutable? Just tested a tiny little program to be sure:
```
#include <iostream>

class M
{
public:
    M(int f) : foo_(f) {}
    int& getFoo() const { return foo_; }
private:
    mutable int foo_;
};

int main()
{
    M m(7);
    const int foo1 = m.getFoo();
    m.getFoo() = 8;
    const int foo2 = m.getFoo();
    std::cout << foo1 << ' ' << foo2 << std::endl;
}
```
The program compiles and runs as expected.